vase-server/models/userinfo.go

```diff
@@ -0,0 +1,13 @@
+package models
+
+import (
+	"gorm.io/gorm"
+)
+
+type UserInfo struct {
```

Comment:
some modeling comments:

Address
- should people be allowed multiple? if so needs to be a table
- will people only have a lotus address? otherwise should be namespaced as such (like XpiAddress)
- should have a unique index

Lat/Long
- ideally in a `Location` struct

Token
- ideally named `PushToken` for better understanding